tests/bootstrap.php

```diff
@@ -6,9 +6,6 @@
  * @subpackage Tests
  */
 
-// Map path to phpunit-polyfills for WordPress >= 5.9.
-const WP_TESTS_PHPUNIT_POLYFILLS_PATH = __DIR__ . '/../vendor/yoast/phpunit-polyfills'; // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
```

Comment:
I saw this was downgraded, I'm assuming it is used elsewhere so that is why the composer package is being kept rather than removed?

Reply:
It's required for the tests in `wordpress-develop`, but apparently not needed in our bootstrap.